docs/theming.md

```diff
@@ -55,3 +55,22 @@ const InvertColors = ({children}) => (
   </ThemeProvider>
 )
 ```
+
+## Getting the theme outside styled components
+
+If you ever need to get the current `theme` outside styled components (e.g. inside render functions), you can use the `withTheme` Higher Order Component:
```

Comment:
"e.g. inside render functions" doesn't make much sense to me, what about "e.g. inside big components"?